instructions.html

```diff
@@ -7,67 +7,21 @@ <h3>Browse Online</h3>
 
 <ul>
   <li><a
-    href="http://code.google.com/p/google-api-java-client/source/browse/">Browse
+    href="https://github.com/google/google-api-java-client">Browse
   Source</a></li>
 </ul>
 
 <h3>Checkout Instructions</h3>
 
-<p><b>Prerequisites:</b> install <a href="http://java.com">Java 6</a>, <a
-  href="http://mercurial.selenic.com/">Mercurial</a> and <a
+<p><b>Prerequisites:</b> install
+  <a href="http://www.oracle.com/technetwork/java/javase/downloads/index.html">
+    Java SE 8</a>, <a href="https://git-scm.com">Git</a> and <a
```

Comment:
Wait I thought Java 7 was the minimum...

Reply:
It is. Maybe _Prerequisites_ is the wrong word to use here. I figured if a developer is following these instructions to install a jdk, directing them to download 8 from that page is a lot simpler than anything else.

Reply:
Well we should change the wording: Prerequisite is Java 7, but Java 8 is recommended